packages/medusa/src/api-v2/admin/regions/[id]/route.ts

```diff
@@ -17,3 +22,41 @@ export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
 
   res.status(200).json({ region })
 }
+
+export const POST = async (req: MedusaRequest, res: MedusaResponse) => {
+  const updateRegions = updateRegionsWorkflow(req.scope)
```

Comment:
suggestion: should we pass the container as part of the run to remove this extra step?